SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -36,43 +40,15 @@ void USpatialWorkerConnection::FinishDestroy()
 
 void USpatialWorkerConnection::DestroyConnection()
 {
-	Stop(); // Stop OpsProcessingThread
-	if (OpsProcessingThread != nullptr)
-	{
-		OpsProcessingThread->WaitForCompletion();
-		OpsProcessingThread = nullptr;
-	}
-
-	if (WorkerConnection)
-	{
-		AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [WorkerConnection = WorkerConnection]
-		{
-			Worker_Connection_Destroy(WorkerConnection);
-		});
-
-		WorkerConnection = nullptr;
-	}
-
-	if (WorkerLocator)
-	{
-		AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [WorkerLocator = WorkerLocator]
-		{
-			Worker_Alpha_Locator_Destroy(WorkerLocator);
-		});
-
-		WorkerLocator = nullptr;
-	}
-
 	bIsConnected = false;
 	NextRequestId = 0;
-	KeepRunning.AtomicSet(true);
 }
 
 void USpatialWorkerConnection::Connect(bool bInitAsClient)
 {
+#include "Interop/GlobalStateManager.h"
```

Comment:
This include should not be here, I am pretty sure.

Reply:
Lol. I'm not sure what happened there.

Reply:
Although technically correct :P.